alf/environments/torch_wrappers.py

```diff
@@ -128,6 +130,59 @@ def duration(self):
         return self._duration
 
 
+def _spec_channel_transpose(spec):
+    """Transpose the third (channel) dimension of the spec to the first.
+    """
+    assert isinstance(spec, ts.TensorSpec)
+    if len(spec.shape) == 3:
+        if spec.is_bounded():
+            return ts.BoundedTensorSpec(spec.shape[::-1], spec.dtype,
+                                        spec.minimum, spec.maximum)
+        else:
+            return ts.TensorSpec(spec.shape[::-1])
```

Comment:
::-1 is not correct, because this will swap height and width

Reply:
nice catch, corrected.